examples/model_selection/grid_search_text_feature_extraction.py

```diff
@@ -102,12 +105,13 @@
     # 'clf__max_iter': (10, 50, 80),
 }
 
+# %%
+# Find the best parameters for both the feature extraction and the
+# classifier
 if __name__ == "__main__":
     # multiprocessing requires the fork to happen in a __main__ protected
     # block
```

Comment:
Could you remove this `if` statement and unindent the code below? Otherwise, it looks good.

Reply:
Done!